index.bs

```diff
@@ -4168,6 +4200,800 @@ opened</dfn> steps given |window|, |type| and |message|.
+
+If the cookie value can be represented as a UTF-8 encoded string, the
+<code>value</code> field will be present. Otherwise the <code>binaryValue</code>
+field will be present and consist of an array of integers representing the bytes
+of the cookie value.
+
+<div algorithm>
+To <dfn>get a cookie</dfn> given |stored cookie|:
+
+Note: The definitions of |stored cookie|'s fields are from [[COOKIES]], except
+samesite-flag, which is from [[SAME-SITE-COOKIES]].
+
+1. Let |name| be the result of [=UTF-8 decode=] with |stored cookie|'s name field.
+
+1. Let |utf8 decoded value| be the result of [=UTF-8 decode without BOM or fail=]
+   with |stored cookie|'s value.
+
+1. If |utf8 decoded value| is failure, then:
+
+    1. Let |value| be null and |binary value| be an empty list.
+
+    1. For each |byte| in |stored cookie|'s value:
+
+      1. Append the [=byte/value=] of |byte| to |binary value|.
+
+   Otherwise: Let |value| be |utf8 decoded value| and |binary value| be null.
+
+1. Let |domain| be |stored cookie|'s domain field.
+
+1. Let |path| be |stored cookie|'s path field.
+
+1. Let |expires| be |stored cookie|'s expires field represented as a unix
+   timestamp, if set, or null otherwise.
+
+1. Let |size| be the byte length of the result of serializing |stored cookie|
+   as it would be represented in a <code>Cookie</code> header.
+
+1. Let |http only| be true if |stored cookie|'s http-only-flag is true, or false
+   otherwise.
+
+1. Let |secure| be true if |stored cookie|'s secure-only-flag is true, or false
+   otherwise.
+
+1. Let |same site| be "<code>none</code>" if |stored cookie|'s samesite-flag is
+   "<code>None</code>", "<code>lax</code>" if it is "<code>Lax</code>, or
+   "<code>strict</code>" if it is "<code>Strict</code>".
+
+1. Return a map matching the <code>network.Cookie</code> production,
+   with the <code>name</code> field set to |name|, the <code>value</code> field
+   set to |value| if it's not null or omitted otherwise, the
+   <code>binaryValue</code> field set to |binary value| if it's not null or
+   omitted otherwise, the <code>domain</code> field set to |domain|, the
+   <code>path</code> field set to |path|, the <code>expires</code> field set to
+   |expires| if it's not null, or omitted otherwise, the <code>size</code> field
+   set to |size|, the <code>httpOnly</code> field set to |http only|, the
+   <code>secure</code> field set to |secure|, and the <code>sameSite</code>
+   field set to |same site|.
+
+</div>
+
+#### The network.FetchTimingInfo type #### {#type-network-FetchTimingInfo}
+
+[=Remote end definition=] and [=local end definition=]
+
+<pre class="cddl local-cddl">
+network.FetchTimingInfo = {
+    timeOrigin: float,
+    requestTime: float,
+    redirectStart: float,
+    redirectEnd: float,
+    fetchStart: float,
+    dnsStart: float,
+    dnsEnd: float,
+    connectStart: float,
+    connectEnd: float,
+    tlsStart: float,
+    <!-- tlsEnd: float this should be the same as connectEnd -->
```

Comment:
There's a [WPT test](https://github.com/web-platform-tests/wpt/blob/707500a0474977b3eea2873288d9c395cd123626/webdriver/tests/bidi/network/__init__.py#L48) that checks for `tlsEnd`, however it is [not currently](https://w3c.github.io/webdriver-bidi/#type-network-FetchTimingInfo) part of the spec.

We should fix either side.

What is the correct one @jgraham ?

Reply:
I am going to remove it from WPT: https://github.com/web-platform-tests/wpt/pull/40223

Reply:
Given that the PR got merged it looks like this is done now.

Reply:
Yes, resolving. Thanks